src/code/z_en_item00.c

```diff
@@ -1,7 +1,6 @@
 #include "global.h"
 #include "overlays/actors/ovl_En_Elf/z_en_elf.h"
 #include "overlays/effects/ovl_Effect_Ss_Dead_Sound/z_eff_ss_dead_sound.h"
-#include "objects/gameplay_keep/gameplay_keep.h"
 
```

Comment:
theres definitely things from gameplay keep in here. how is this compiling?

Reply:
The gameplay keep header is in the .data.c file

Reply:
ah. as a seperate point from that, I was going to say that imo a data.c isnt really needed here